src/pages/RFOX/components/Claim/ClaimStatus.tsx

```diff
@@ -91,9 +95,20 @@ export const ClaimStatus: React.FC<Pick<ClaimRouteProps, 'headerComponent'> & Cl
     }
   }, [claimAsset, txStatus, claimAmountCryptoPrecision])
 
+  const { data: safeTx } = useSafeTxQuery({
+    maybeSafeTxHash: txId ?? undefined,
+    accountId,
+  })
+
   const txLink = useMemo(
-    () => getTxLink({ txId, defaultExplorerBaseUrl: claimAsset?.explorerTxLink ?? '' }),
-    [claimAsset?.explorerTxLink, txId],
+    () =>
+      getTxLink({
```

Comment:
Don't forget to remove TODO of `getTxLink` - see https://github.com/shapeshift/web/pull/7506#discussion_r1725550786